integration-tests/container/processes-listening-on-ports/plop.py

```diff
@@ -15,7 +15,7 @@ def open_port(port: int):
     s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
 
     try:
-        s.bind((socket.gethostname(), port))
+        s.bind(('0.0.0.0', port))
```

Comment:
This was needed because `socket.gethostname()` was failing on Garden, binding to the wildcard IP is enough for the tests.